frontend/src/components/Sidebar.tsx

```diff
@@ -600,3 +606,78 @@ export function useSidebarItem(itemName: string | null) {
   []);
 }
 
+function searchNameInSubList(sublist : SidebarEntry['subList'], name: string): boolean {
+  if (!sublist){
+    return false;
+  }
+  for (let i = 0;i < sublist.length;i++) {
+    if (sublist[i].name === name){
+      return true;
+    }
+  }
+  return false;
+}
+
+function findParentOfSubList(list: SidebarEntry[], name: string | null): SidebarEntry | null{
+  if (!name) {
+    return null;
+  }
+
+  let parent = null;
+  for (let i = 0;i < list.length;i++) {
+    if (searchNameInSubList(list[i].subList, name)) {
+      parent = list[i];
+    }
+  }
+  return parent;
+}
+
+export function NavigationTabs() {
+  const history = useHistory();
+  const sidebar = useTypedSelector(state => state.ui.sidebar);
+  if (sidebar.isSidebarOpen) {
+    return null;
+  }
+  let isDefaultSelectedInTabs = false;
+  const listItems = prepareRoutes();
+  let navigationItem = listItems.find((item) => item.name === sidebar.selected);
+  let defaultIndex = 0;
+  if (!navigationItem) {
+    const parent = findParentOfSubList(listItems, sidebar.selected);
+    if (!parent) {
+      return null;
+    }
+    navigationItem = parent;
+    isDefaultSelectedInTabs = true;
+  }
+  const subList = navigationItem.subList;
+  if (!subList) {
+    return null;
+  }
+  function tabChangeHandler(index: number) {
+    if (!subList) {
+      return;
+    }
+    let pathname;
+    const url = subList[index].url;
+    if (url) {
+      pathname = generatePath(getClusterPrefixedPath(url), {cluster: getCluster()!});
+    } else {
+      pathname = createRouteURL(subList[index].name);
+    }
+    history.push({pathname});
+  }
+  const tabRoutes = subList.map((item: any) => {
+    return {'label': item.label, component: <></>};
+  });
+
+  defaultIndex = subList.findIndex((item) => item.name === sidebar.selected);
+  return (
+    <Tabs
+      tabs={tabRoutes}
+      isDefaultSelectedRequired={isDefaultSelectedInTabs}
+      resetState={sidebar.isIconClicked}
+      onTabChanged={(index) => {tabChangeHandler(index);}}
+      defaultIndex={defaultIndex}
```

Comment:
Right. I think this is too complicated. I'd rather just pass it the `tabIndex` to determine the default tab, and then if the tabs are to be hidden, you can return `null` here, so the next time it's rebuilt, the default tab will be set correctly.
You can also move the divider here so it'll not be shown when the tabs should be hidden.